docker build error

run docker build -t xxxx .
with dockerfile
`ARG FROM=ghcr.io/emqx/emqx-builder/5.0-31:1.13.4-24.3.4.2-2-debian11
FROM ${FROM}
COPY . /emqtt_bench
WORKDIR /emqtt_bench
RUN make compile
ENTRYPOINT ["/emqtt_bench/emqtt_bench"]
CMD [""]`
comes error
`escript: exception error: undefined function rebar3:main/1
  in function  escript:run/2 (escript.erl, line 750)
  in call from escript:start/1 (escript.erl, line 277)
  in call from init:start_em/1 
  in call from init:do_boot/3 
make: *** [Makefile:12: compile] Error 127
`
how can i fix it?

--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -11,7 +11,7 @@ jobs:
         otp:
           - 25.3.2-2
         elixir:
-          - 1.14.5
+          - 1.15.7
     steps:
       - uses: actions/checkout@v3
         with:
@@ -39,7 +39,7 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           build-args: |
-            FROM=ghcr.io/emqx/emqx-builder/5.2-3:${{ matrix.elixir }}-${{ matrix.otp }}-debian11
+            FROM=ghcr.io/emqx/emqx-builder/5.2-7:${{ matrix.elixir }}-${{ matrix.otp }}-debian11
 
   linux:
     runs-on: ubuntu-latest
@@ -49,7 +49,7 @@ jobs:
         otp:
           - 25.3.2-2
         elixir:
-          - 1.14.5
+          - 1.15.7
         quic_support:
           - true
           - false
@@ -57,7 +57,7 @@ jobs:
           - ubuntu22.04
 
     container:
-      image: ghcr.io/emqx/emqx-builder/5.2-3:${{ matrix.elixir }}-${{ matrix.otp }}-${{ matrix.os }}
+      image: ghcr.io/emqx/emqx-builder/5.2-7:${{ matrix.elixir }}-${{ matrix.otp }}-${{ matrix.os }}
 
     steps:
       - uses: actions/checkout@v3
@@ -90,6 +90,7 @@ jobs:
         macos:
           - macos-12-arm64
           - macos-12
+          - macos-13
         otp:
           - 25.3.2-2
 

--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -20,9 +20,9 @@ jobs:
       fail-fast: false
       matrix:
         otp:
-          - 25.3.2-1
+          - 25.3.2-2
         elixir:
-          - 1.14.5
+          - 1.15.7
     steps:
       - uses: actions/checkout@v3
         with:
@@ -52,7 +52,7 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           build-args: |
-            FROM=ghcr.io/emqx/emqx-builder/5.1-3:${{ matrix.elixir }}-${{ matrix.otp }}-debian11
+            FROM=ghcr.io/emqx/emqx-builder/5.2-7:${{ matrix.elixir }}-${{ matrix.otp }}-debian11
 
   linux:
     runs-on: ubuntu-latest
@@ -60,9 +60,9 @@ jobs:
       fail-fast: false
       matrix:
         otp:
-          - 25.3.2-1
+          - 25.3.2-2
         elixir:
-          - 1.14.5
+          - 1.15.7
         quic_support:
           - true
           - false
@@ -81,7 +81,7 @@ jobs:
           - amzn2
           - amzn2023
     container:
-      image: ghcr.io/emqx/emqx-builder/5.1-3:${{ matrix.elixir }}-${{ matrix.otp }}-${{ matrix.os }}
+      image: ghcr.io/emqx/emqx-builder/5.2-7:${{ matrix.elixir }}-${{ matrix.otp }}-${{ matrix.os }}
 
     steps:
       - uses: actions/checkout@v3
@@ -115,8 +115,9 @@ jobs:
         macos:
           - macos-12-arm64
           - macos-12
+          - macos-13
         otp:
-          - 25.3.2-1
+          - 25.3.2-2
 
     runs-on: ${{ matrix.macos }}
 

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-ARG FROM=ghcr.io/emqx/emqx-builder/5.0-31:1.13.4-24.3.4.2-2-debian11
+ARG FROM=ghcr.io/emqx/emqx-builder/5.2-7:1.15.7-25.3.2-2-debian11
 FROM ${FROM}
 COPY . /emqtt_bench
 WORKDIR /emqtt_bench

